Update connect-gitlab.md (#7216)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/git/connect-gitlab.md

* Push rules aren't supported
* Only the user who originally connected the repository can commit changes

To enable multi-user commit access, you’ll need an Enterprise plan, which supports deploy tokens that aren't tied to a specific user.

:::info
When configuring the repository in <Constant name="cloud" />, GitLab automatically:
- Registers a webhook that triggers pipeline jobs in <Constant name="cloud" />.
- Creates a [project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) in your GitLab repository, which sends the job run status back to GitLab using the <Constant name="cloud" /> API for CI jobs. <Constant name="cloud" /> automatically refreshes this token for you, which means you never have to manually rotate it. Check out the [troubleshooting](#troubleshooting) section for more information.

:::

Depending on your plan, use these steps to integrate GitLab in <Constant name="cloud" />:
- the Developer or Team plan, read these [instructions](#for-dbt-cloud-developer-and-team-tiers).
- the Enterprise plan, jump ahead to these [instructions](#for-the-dbt-cloud-enterprise-tier).

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

:::info
When configuring the repository in <Constant name="cloud" />, GitLab automatically:
- Registers a webhook that triggers pipeline jobs in <Constant name="cloud" />.
- Creates a [project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) in your GitLab repository, which sends the job run status back to GitLab using the <Constant name="cloud" /> API for CI jobs. <Constant name="cloud" /> automatically refreshes this token for you, which means you never have to manually rotate it. Check out the [troubleshooting](#troubleshooting) section for more information.

:::

Depending on your plan, use these steps to integrate GitLab in <Constant name="cloud" />:
- the Developer or Starter plan, read these [instructions](#for-dbt-developer-and-starter-plans).

> - the Enterprise or Enterprise+ plan, jump ahead to these [instructions](#for-the-dbt-enterprise-plan).

Reviewer: ```suggestion
- the Enterprise or Enterprise+ plan, jump ahead to these [instructions](#for-the-dbt-enterprise-plans).
```

:::info
When configuring the repository in <Constant name="cloud" />, GitLab automatically:
- Registers a webhook that triggers pipeline jobs in <Constant name="cloud" />.
- Creates a [project access token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) in your GitLab repository, which sends the job run status back to GitLab using the <Constant name="cloud" /> API for CI jobs. <Constant name="cloud" /> automatically refreshes this token for you, which means you never have to manually rotate it. Check out the [troubleshooting](#troubleshooting) section for more information.

:::

Depending on your plan, use these steps to integrate GitLab in <Constant name="cloud" />:
- the Developer or Team plan, read these [instructions](#for-dbt-cloud-developer-and-team-tiers).
- the Enterprise plan, jump ahead to these [instructions](#for-the-dbt-cloud-enterprise-tier).

## For dbt Cloud Developer and Team tiers

> Before you can work with GitLab repositories in <Constant name="cloud" />, you’ll need to connect your GitLab account to your user profile. This allows <Constant name="cloud" /> to authenticate your actions when interacting with Git repositories. Make sure to read about [limitations](#limitations) of the Team plan before you connect your account.

Reviewer: Are these only Team plan limitations or also developer plan?

> Before you can work with GitLab repositories in <Constant name="cloud" />, you’ll need to connect your GitLab account to your user profile. This allows <Constant name="cloud" /> to authenticate your actions when interacting with Git repositories. Make sure to read about [limitations](#limitations) of the Team plan before you connect your account.

Reviewer: ```suggestion
Before you can work with GitLab repositories in <Constant name="cloud" />, you’ll need to connect your GitLab account to your user profile. This allows <Constant name="cloud" /> to authenticate your actions when interacting with Git repositories. Make sure to read about [limitations](#limitations) of the Team and Developer plans before you connect your account.
```

Once you've accepted, you should be redirected back to <Constant name="cloud" />, and you'll see that your account has been linked to your profile.

### Limitations

> 

Reviewer: ```suggestion
Developer and Team plans only allow single-user commit access.
```

> Because <Constant name="cloud" /> Team plans use a single GitLab deploy token that's tied to the user who connected the repo, they are limited in the following ways when using GitLab:

Reviewer: Same as above, should this say Team and Developer plans?

Reviewer: No dev plans only have a single user.

> Because <Constant name="cloud" /> Team plans use a single GitLab deploy token that's tied to the user who connected the repo, they are limited in the following ways when using GitLab:

Reviewer: ```suggestion
<Constant name="cloud" /> Team and Developer plans use a single GitLab deploy token created by the first user who connects the repository, which means:
* All Git operations (like commits and pushes) from the <Constant name="ide" /> appear as coming from the same deploy token.
* GitLab push rules may reject pushes made through <Constant name="cloud" />, particularly when multiple users are committing via the same deploy token. 
 
```

> * Push rules aren't supported

Reviewer: ```suggestion
```

> * All Git operations (like commits and pushes) from the <Constant name="ide" /> appear as coming from the same deploy token.

Reviewer: ```suggestion
* All Git operations (like commits and pushes) from the <Constant name="cloud_ide" /> appear as coming from the same deploy token.
```

> * Only the user who originally connected the repository can commit changes

Reviewer: ```suggestion
```

> To enable multi-user commit access, you’ll need an Enterprise plan, which supports deploy tokens that aren't tied to a specific user.

Reviewer: ```suggestion
To support advanced Git workflows and multi-user commit behavior, upgrade to the Enterprise plan, which provides more flexible Git authentication strategies.
```

Once you've accepted, you should be redirected back to <Constant name="cloud" />, and you'll see that your account has been linked to your profile.

### Limitations

<Constant name="cloud" /> Team and Developer plans use a single GitLab deploy token created by the first user who connects the repository, which means:
* All Git operations (like commits and pushes) from the <Constant name="ide" /> appear as coming from the same deploy token.
* GitLab push rules may reject pushes made through <Constant name="cloud" />, particularly when multiple users are committing via the same deploy token. 

>  

Reviewer: ```suggestion
```

Once you've accepted, you should be redirected back to <Constant name="cloud" />, and you'll see that your account has been linked to your profile.

### Limitations

<Constant name="cloud" /> Team and Developer plans use a single GitLab deploy token created by the first user who connects the repository, which means:
* All Git operations (like commits and pushes) from the <Constant name="cloud_ide" /> appear as coming from the same deploy token.
* GitLab push rules may reject pushes made through <Constant name="cloud" />, particularly when multiple users are committing via the same deploy token. 

To support advanced Git workflows and multi-user commit behavior, upgrade to the Enterprise plan, which provides more flexible Git authentication strategies.

> ## For the dbt Enterprise plan

Reviewer: ```suggestion
## For the dbt Enterprise plans
```
